src/components/pages/bc-new-wallet-link.ts

```diff
@@ -0,0 +1,105 @@
+import {customElement} from 'lit/decorators.js';
+import {BitcoinConnectElement} from '../BitcoinConnectElement';
+import {withTwind} from '../twind/withTwind';
+import {html} from 'lit';
+import {classes} from '../css/classes';
+import '../internal/bci-button';
+import {albyIcon} from '../icons/connectors/albyIcon';
+
+@customElement('bc-new-wallet-link')
+export class NewWalletPage extends withTwind()(BitcoinConnectElement) {
+  override render() {
+    return html`<div>
+      <bc-navbar
+        class="flex w-full"
+        heading="Get a bitcoin lightning wallet"
+      ></bc-navbar>
+
+      <div class="flex gap-4 w-full my-6 px-8">
```

Comment:
can this and the div below be merged into one? it seems the second is unnecessary